src/components/fields/image-upload/image-manager/image-manager.ts

```diff
@@ -190,10 +191,11 @@ export const ImageManager = (props: IProps) => {
 
 		setValue(
 			id,
-			uploadedImages.map(({ id, dataURL, drawingDataURL, name, uploadResponse }) => ({
+			uploadedImages.map(({ id, dataURL, drawingDataURL, name, metadata, uploadResponse }) => ({
 				fileId: id,
 				fileName: name,
 				dataURL: drawingDataURL || dataURL,
+				metadata,
```

Comment:
can we have a new option in the schema to control whether to send the metadata? I want to avoid causing controversies by sending potentially sensitive data for other consumers of this component

Reply:
iinw this data can be extracted from the base64. so if that data is available to the consumer, then this metadata should not be any more sensitive.
@namlh023 is that correct?

Reply:
As long as we return the dataURL, the consumer can extract it to access the metadata. I'm just unsure about how much information they can retrieve from it.
![Uploading Screenshot 2024-10-25 at 14.44.47.png…]()

Reply:
Can confirm that consumers can access all the image metadata from the data URL we provide (including GPS and DateTime). Therefore, adding a new option may not be necessary.

Reply:
ok my concern is that we are explicitly extracting the metadata now and putting them out in plain text as part of the submitted values.

however I think we can go ahead to do this first since the data is in the base64 payload in the first place, perhaps the question is whether we should be able to strip the metadata instead of not extracting the metadata. we will tackle this when there is a request to do it.